apps/CA.pl.in

```diff
@@ -59,21 +77,30 @@ sub crl_reason_ok
     exit 1;
 }
 
-# Copy a PEM-format file; return like exit status (zero means ok)
+# PEM-format file manipulation - copy, concatenate, remove comments from;
+# return like exit status (zero means ok)
 sub copy_pemfile
 {
-    my ($infile, $outfile, $bound) = @_;
+    my ($infile, $outfile, $bound, $outmode) = @_;
+    my $onthefly = 0;
     my $found = 0;
 
+    $outmode = ">" unless $outmode;
+    if ($infile eq $outfile) {
+        $onthefly = 1;
+        copy($infile,$infile."_tmp") || die "Cannot copy $infile, $!";
```

Comment:
That was, considering `$outmode` is always `">"`.  I totally spaced on the fact that `$outmode` could be `">>"`...  so, never mind